app/utils/scripts/parseLocation.ts

```diff
@@ -0,0 +1,10 @@
+export const parseLocation = (location: string) => {
```

Comment:
no need to make changes now, but we can name it app/untils/common.ts or lib.ts 

And put all common functions to the same file, if the functions are all short.